game/src/js/scenes/GamepadScene.js

```diff
@@ -0,0 +1,91 @@
+import BaseScene from '../scenes/BaseScene';
+
+export default class GamepadScene extends BaseScene {
+  constructor() {
+    super({ key: 'GamepadScene', active: true });
```

Comment:
Can we activate this scene only when we need it?

Reply:
I'm not sure, I was looking for a way to set the active bit on a scene but couldn't find it, so I followed how we did it with the old modals and used a visible flag.

Reply:
I modified the scene to use start/stop so it is only active when needed, I think that's what we were looking for?